handlers/src/main/java/com/nextdoor/bender/handler/dynamodb/DynamodbEventSerializer.java

```diff
@@ -0,0 +1,137 @@
+package com.nextdoor.bender.handler.dynamodb;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.StreamRecord;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+import com.google.gson.stream.JsonWriter;
+
+public class DynamodbEventSerializer {
```

Comment:
Let's change this to use https://stackoverflow.com/a/43918963/384973. Note be sure to use a static instance of the serializer and `Gson` when doing so.